Transmission.xcodeproj/project.pbxproj

```diff
@@ -4079,9 +4068,11 @@
 		4DF0C599089918A300DD8943 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Transmission supports 10.14.6+, right? I don't think we need to enable this setting because Swift libraries are already bundled in macOS since 10.14.4 and bundling them once again inside the app will only increase the size of the app

Reply:
Ah, interesting. This setting was auto-added when I first created a swift file. I'm not so sure if it makes really a difference to have it or not, but I can try.
Possibly, you mean 10.14.4 regarding what is written in https://www.swift.org/blog/abi-stability-and-apple/ ?